diana.navasardyan/repo/Lesson_3/exercise_17/exercise_17.py

```diff
@@ -0,0 +1,28 @@
+inp_number1 = int(input("Enter a number: "))
+inp_number2 =  int(input("Enter a number: "))
+inp_number3 =  int(input("Enter a number: "))
+
+sum_numbers = inp_number1 + inp_number2 + inp_number3
+
+averege_numbers = sum_numbers / 3
+
+product_numbers = inp_number1 * inp_number2 * inp_number3
+#largest_number
+if inp_number2 < inp_number1 > inp_number3:
```

Comment:
LW: Use fewer comparisons.

Hint. The first number can be assigned to a variable like "largest".